packages/tokens/build.js

```diff
@@ -46,6 +76,36 @@ StyleDictionary.registerFormat({
   },
 })
 
+/** Custom format for themes. Filters light and dark mode themes into properties
+ *  and exports them as a single themes object */
+StyleDictionary.registerFormat({
+  name: 'javascript/es6/vads-colors-themes',
+  formatter: function (dictionary) {
+    const light = dictionary.allProperties
+      .filter(filterLight)
+      .reduce(stripModeReducer, {})
+
+    const dark = dictionary.allProperties
+      .filter(filterDark)
+      .reduce(stripModeReducer, {})
+
+    return `export const themes = {
+  light: ${JSON.stringify(sortTokensByKey(light), null, 4)},
+  dark: ${JSON.stringify(sortTokensByKey(dark), null, 4)}
+    }`
+  },
+})
+
+/** Custom format to generate index js with exports */
+StyleDictionary.registerFormat({
+  name: 'javascript/es6/vads-module-export',
+  formatter: function () {
+    return `export { Colors } from './colors'
+export { themes } from './themes'
```

Comment:
@TimRoe Do you have any strong feelings on whether these exports should be in camelCase or PascalCase? 

I know we started out with Colors in PascalCase but after searching around a bit, seems PascalCase is typically used for classes, interfaces, type aliases, enums, and namespaces.

Just want to get `Colors` and `themes` in sync. The app isn't using `Colors` directly yet so changing it up should be inconsequential. Just need to update it throughout our components, which should be minimized anyway since we're moving to using themes instead.

Reply:
Don't have strong feelings. Thought in the past when I looked PascalCase was more common, but, poking around a little, I'm seeing camelCase.

Agreed with trying to have some level of consistency among building block type things. 

Would note that when doing the icon script I discovered that the icon names (a building block type thing) can't be camelCase without causing issues because VADS have icon names like "delete" and a couple others that are reserved keywords in JS. Don't think that should be a problem with token sets though.

Reply:
Updates to `colors` to match `themes`